### ISSUE TITLE
Hover/scroll events when not focused

### ISSUE THREAD

**Issue by PazerOP** [PazerOP/tf2_bot_detector#28]
Normal Windows applications allow you to scroll or trigger tooltips even with the window doesn't have keyboard focus. Currently this doesn't work in TF2BD.

**Comment by ClusterConsultant** [comment]
This was fixed at some point
